src/iop/demosaic.c

```diff
@@ -982,7 +990,741 @@ static void xtrans_markesteijn_interpolate(float *out, const float *const in,
   }
   dt_free_align(all_buffers);
 }
+#undef TS
+
+#define TS 122
+/*
+  Ingo Liebhardt 's  frequency domain chroma approach for Fuji X-Trans sensors
+ */
+static void xtrans_fdc_interpolate(float *out, const float *const in,
```

Comment:
I really do hope you formatted this with `clang-format` with our `.clang-format`.